### PR TITLE
Add message content intent

### PR DESCRIPTION
**Pull request by Lulalaby**
Here again, this time working.

Sorry for the mess <3

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -145,7 +160,7 @@ function App() {
             {
               Object.keys(intents).map(key => 
                 <FormControlLabel
-                  control={<Checkbox checked={state[key]} onChange={handleChange} name={key} color='default' disabled={key === "GUILD_PRESENCES" ? !state.privilegedIntents.presence : key === "GUILD_MEMBERS" && !state.privilegedIntents.guildMembers} indeterminate={key === "GUILD_PRESENCES" ? !state.privilegedIntents.presence : key === "GUILD_MEMBERS" && !state.privilegedIntents.guildMembers} />}
+                  control={<Checkbox checked={state[key]} onChange={handleChange} name={key} color='default' disabled={key === "GUILD_PRESENCES" ? !state.privilegedIntents.presence : key === "GUILD_MEMBERS" ? !state.privilegedIntents.guildMembers : key === "MESSAGE_CONTENT" && !state.privilegedIntents.messageContent} indeterminate={key === "GUILD_PRESENCES" ? !state.privilegedIntents.presence : key === "GUILD_MEMBERS" ? !state.privilegedIntents.guildMembers : key === "MESSAGE_CONTENT" && !state.privilegedIntents.messageContent} />}
```
Comment (Larkooo) [inline src/App.tsx:163]: make this a bit more readable

File: src/Intents.tsx [hunk 2]
```diff
@@ -110,6 +110,10 @@ const intents : {[key: string]: [string[], number]} = {
   DIRECT_MESSAGE_TYPING: [[
     "TYPING_START"
   ], 14],
+    
+  MESSAGE_CONTENT: [[
+      "MESSAGE_CONTENT"
```
Comment (Larkooo) [inline src/Intents.tsx]: this should be empty. MESSAGE_CONTENT doesnt have any events related to it https://discord.com/developers/docs/topics/gateway#list-of-intents

File: src/App.tsx (hunk 1 above)
Comment (Lulalaby) [reply src/App.tsx:163]: Could you do this please? TS is not my best language. I struggle with it. And I'm out of order rn (ill)

File: src/Intents.tsx [hunk 3]
```diff
@@ -110,6 +110,10 @@ const intents : {[key: string]: [string[], number]} = {
...
+  ], 15],
```
Comment (Lulalaby) [inline src/Intents.tsx:116]:
```suggestion
  MESSAGE_CONTENT: [[], 15],
```

**Pull request merged**